### PR TITLE
[Workers AI] Adds AutoTrain tutorial

### PR DESCRIPTION
**Pull request by craigsdennis**
This adds a HuggingFace AutoTrain tutorial

### CODE REVIEW

**Comment by craigsdennis** [comment]
I plan to follow this up with a video and add it to YouTube and the docs

File: content/workers-ai/tutorials/fine-tune-models-with-autotrain.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+Modify the following fields
+
+- **project_name**: Choose a descriptive name for you to remember later
+- **model_name**: Choose from the one of the official HuggingFace base models that we support:
+  - `mistralai/Mistral-7B-Instruct-v0.2`
+  - `google/gemma-2b-it`
+  - `google/gemma-7b-it`
+  - `meta-llama/llama-2-7b-chat-hf`
+
+### Optional: Push to Hub?
```
Comment (markdembo) [inline content/workers-ai/tutorials/fine-tune-models-with-autotrain.md:63]: Remove ?

File: content/workers-ai/tutorials/fine-tune-models-with-autotrain.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+
+Although not required to use AutoTrain, creating a HuggingFace account will help you keep your finetune artifacts in a handy repository for you to refer to later.
```
Comment (markdembo) [inline content/workers-ai/tutorials/fine-tune-models-with-autotrain.md:65]: Add link to account creation

File: content/workers-ai/tutorials/fine-tune-models-with-autotrain.md [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+
+{{<Aside type="warning">}}
+At the time of this writing, changing the quantization field breaks the code generation. You may need to edit the code and put quotes around the value.
+
+Change the line that says `quantization = none` to `quantization = "none"`.
+{{</Aside>}}
+
+## 3. Upload your CSV file to the Notebook
+
+Notebooks have a folder structure which you can access by clicking the folder icon on the left hand nav bar.
```
Comment (markdembo) [inline content/workers-ai/tutorials/fine-tune-models-with-autotrain.md:86]: nit: navigation bar

File: content/workers-ai/tutorials/fine-tune-models-with-autotrain.md [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+
+## 7. Upload the Fine Tune to your Cloudflare Account
+
+Now that you have your files, you can add them to your account.
+
+You can either use the [REST API or Wrangler](/workers-ai/fine-tunes/loras/).
+
+## 8. Use your Fine Tune in your Generations
+
+After you have your new fine tune all set up, you are ready to [put it to use in your inference requests](http://localhost:5173/workers-ai/fine-tunes/loras/#running-inference-with-loras).
```
Comment (markdembo) [inline content/workers-ai/tutorials/fine-tune-models-with-autotrain.md]: Needs fix, will do it

**Review by markdembo — CHANGES_REQUESTED** [review]
Approved, but improvements needed.

**Review by markdembo — APPROVED** [review]

**Review by angelampcosta — APPROVED** [review]

**Pull request merged**